istioctl/cmd/proxyconfig.go

```diff
@@ -946,6 +946,77 @@ func secretConfigCmd() *cobra.Command {
 	return secretConfigCmd
 }
 
+func rootCACompareConfigCmd() *cobra.Command {
+	var podName1, podName2, podNamespace1, podNamespace2 string
+
+	rootCACompareConfigCmd := &cobra.Command{
+		Use:   "rootca-comp [pod/]<name-1>[.<namespace-1>] [pod/]<name-2>[.<namespace-2>]",
+		Short: "Compare ROOTCA values for given 2 pods",
+		Long:  `Compare ROOTCA values for given 2 pods to check the connectivity availability between them.`,
```

Comment:
Compare ROOTCA values for given 2 pods to check the connectivity  between them

Reply:
thanks @irisdingbj done!